.github/workflows/integration-tests.yml

```diff
@@ -109,6 +109,15 @@ jobs:
           # Run hopper/test_flashattention.py separately to avoid out of gpu memory
           python3 -m pytest -vs hopper/test_flashattention.py
 
+      - name: Run python examples on CUDA
+        # Execute on H100 due to its ~30s speed advantage over A100;
```

Comment:
Nit, H100 has a ~2m speed advantage over CUDA; it's the examples that take ~30s to run.